RELEASE-NOTES.md

```diff
@@ -1,5 +1,39 @@
 # Prefect Release Notes
 
+## Release 2.10.16
+
+### Run `prefect deploy` without a providing flow entrypoint
+
+We're making it easier than ever to deploy your first flow! Previously you needed to have a flow entrypoint in order to deploy a flow. Now, you can run `prefect deploy` and then select which flow you want to deploy as part of the interactive deployment process.
```

Comment:
instead of "and then select which flow.." 

maybe

"and then our interactive deployment CLI will additionally guide you to select a flow"

Reply:
something like that